dev-notes.md

```diff
@@ -24,12 +23,33 @@ Extended support for arbitrary secrets includes:
 
 For more details on the above, see [Arbitrary Secrets Overview](arbitrary-secrets-overview.md).
 
-### Signing Keys
+## Signing Keys
 
 We want our system to include support of signing keys. In this phase, we will include support for basic ECDSA signing key generation and use.
 
 As part of this phase, we will extend our existing system functionalities to include:
 - Remote generation and storage of secrets, without local storage.
 - Generic support for generation and use of signing keys. Uses will include retrieval, import/export, generating signatures, and obtaining audit logs.
 
+## Cryptographic Protocol and Implementation Dependencies
```

Comment:
I assumed that you copied this block as-is from the old location. Are there changes within this that I should look at more carefully?

Reply:
L49-L54.